js/aioseop-helper.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Use for JavaScript compatability by using anonymous functions; which
+ * are loaded into the footer.
+ *
+ * @summary   AIOSEOP Footer JS
+ *
+ * @since     2.4.2
+ */
+// phpcs:disable PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket
```

Comment:
we should add these to the phpcs.xml file

Reply:
Again, that is why it is added. So that it can be recognized later. This isn't a PR with phpcs.xml changes.

Reply:
if the changes are needed as part of this PR and they are included in the js file, I don't see why they cannot be included in the XML file. They should be removed from the js file too

Reply:
I can't just changed the grunt coding standards, when it's not even part of it. Some of it needs discussing before it can be changed.

Reply:
Sure but if you have added this to the JS file means you have already decided to disable that rule from working on that file => no discussion required. Plus if you want to remove these rules from all over the workspace, then they need to be added to the XML file otherwise the current implementation is fine. I'm not sure what your intent is. 

Either ways, you have already decided that these need to be disabled so there isn't much scope left for discussion. Its confusing if you have already added these to the file and then require discussion. If you require a discussion, these should not be added to any code change and instead a new issue should be created. I'm not sure I understand the intent of adding them to a file but not to another file

Reply:
It is not removed from all over the workspace, and this part of the code it is intended; which are also the coding standards we had issues with when reviewing your grunt PR.

https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage

Reply:
this has been discussed on slack so I'm resolving this conversation